rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -141,7 +146,7 @@ def _find_trades(self, all_logs: list[EvmTxReceiptLog]) -> list[CowswapSwapData]
         swap made in the transaction.
         """
         trades = []
-
+        tx_info = TokenEncounterInfo(tx_hash=tx_hash, description='CowSwap trade')
```

Comment:
Put this in the line above and combine with `trades` in the same initialization line